catalog/admin/admin-poller-service/src/main/java/org/codice/ddf/catalog/admin/poller/AdminPollerServiceBean.java

```diff
@@ -248,7 +248,8 @@ protected String getServiceFactoryFilterProperties() {
   }
 
   private String createFilter(String pidKey) {
-    String includes, excludes;
+    String includes;
+    String excludes;
```

Comment:
❓ Any reason we can't declare these inline to get rid of these ?

```java
String includes = (includeAsSource == null || ...
if (excludeAsSource == null || CollectionUtils.isEmpty(excludeAsSource)){
  return String.format("(|%s)", includes);
} else {
  String excludes = excludesAsSource.stream() ...
  return String.format("(&(|%s)(&%s))", includes, excludes);
}
```

Reply:
Nope. Changed.